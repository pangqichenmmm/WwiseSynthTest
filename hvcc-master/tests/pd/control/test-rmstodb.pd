#N canvas 0 26 360 225 10;
#X obj 33 29 loadbang;
#X msg 33 95 -100;
#X msg 138 95 0.3;
#X msg 168 95 0.5;
#X msg 198 95 0.707;
#X msg 240 95 100;
#X msg 272 95 1000;
#X obj 33 59 t b b b b b b b b;
#X obj 33 165 print;
#X msg 68 95 0;
#X msg 98 95 0.001;
#X obj 33 135 rmstodb;
#X obj 113 21 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X connect 0 0 7 0;
#X connect 1 0 11 0;
#X connect 2 0 11 0;
#X connect 3 0 11 0;
#X connect 4 0 11 0;
#X connect 5 0 11 0;
#X connect 6 0 11 0;
#X connect 7 0 1 0;
#X connect 7 1 9 0;
#X connect 7 2 10 0;
#X connect 7 3 2 0;
#X connect 7 4 3 0;
#X connect 7 5 4 0;
#X connect 7 6 5 0;
#X connect 7 7 6 0;
#X connect 9 0 11 0;
#X connect 10 0 11 0;
#X connect 11 0 8 0;
#X connect 12 0 7 0;
