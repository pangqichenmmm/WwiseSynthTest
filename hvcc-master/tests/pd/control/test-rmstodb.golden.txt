[@ 0.000] print: 160
[@ 0.000] print: 140
[@ 0.000] print: 96.9884
[@ 0.000] print: 93.9794
[@ 0.000] print: 89.5424
[@ 0.000] print: 40
[@ 0.000] print: 0
[@ 0.000] print: 0
